Gemfile

```diff
@@ -42,14 +42,14 @@ supports_windows = true
 
 group :development do
   gem 'puppet-lint',                        :require => false
-  gem 'metadata-json-lint',                 :require => false if ! supports_windows || Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.1')
+  gem 'metadata-json-lint',                 :require => false, :platforms => 'ruby'
```

Comment:
I'm not sure I understand this... or the larger conversation. around the Windows guards.

@ferventcoder mentioned guarding because these tools require the `json` gem. Ruby already bundles / ships what is also shipped externally as the JSON gem.  1.9.3 shipped an older JSON 1.5.5, so that's the only reason I've see to require that the gem is pulled down - i.e. to pick up something in the JSON 2+ release for older Rubies.  For newer Rubies, I've found that the shipped JSON is fine already.